scripts/restore

```diff
@@ -44,6 +44,8 @@ ynh_package_install python3-pip python3-virtualenv
 # Restore sources & data
 sudo cp -a ./sources $final_path
 
+# Create user
+useradd $app -d $final_path || echo "User already exists"
```

Comment:
Uuuh shouldnt we use the appropriate helper for that ? :P Or did you just copypasta from the install script ?

Reply:
Just copy past :-)